qutebrowser/app.py

```diff
@@ -814,6 +826,22 @@ def on_focus_object_changed(self, obj):
             log.misc.debug("Focus object changed: {}".format(output))
         self._last_focus_object = output
 
+    @pyqtSlot(Qt.ApplicationState)
+    def on_app_state_changed(self, state):
+        if state != Qt.ApplicationActive:
+            return
+
+        window = objreg.last_focused_window()
+        mainwindow.raise_window(window)
+
+    def event(self, e):
+        if e.type() == QEvent.FileOpen:
+            open_url(e.url(), force_raise=False)
```

Comment:
You should probably `return True` here ([docs](http://doc.qt.io/qt-5/qobject.html#event))